src/mvStackWindow.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "mvToolWindow.h"
+
+class mvStackWindow final : public mvToolWindow
+{
+
+public:
+
+    [[nodiscard]] mvUUID getUUID() const override { return MV_TOOL_STACK_UUID; }
+    [[nodiscard]] const char* getTitle() const override { return "Stack tool"; }
+
+protected:
+    // TODO: Find a way to remove extra "Stack tool" window
```

Comment:
Re TODO - look here on how to do this:
https://github.com/v-ein/DearPyGui-fixes/compare/porting...bugfix/hide-stack-window

Reply:
Really thank you!!